File: services/quarkus-svc/src/main/java/com/perfsim/quarkussvc/domain/Payment.java
package com.perfsim.quarkussvc.domain;

import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import java.time.OffsetDateTime;

@Entity
@Table(name = "payments", schema = "quarkus")
public class Payment extends PanacheEntityBase {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Long id;

    @Column(name = "order_id", nullable = false)
    public Long orderId;

    @Column(name = "customer_id", nullable = false)
    public Long customerId;

    @Column(name = "amount_cents", nullable = false)
    public long amountCents;

    @Column(nullable = false, length = 32)
    public String status = "AUTHORIZED";

    @Column(name = "created_at", nullable = false, insertable = false, updatable = false)
    public OffsetDateTime createdAt;
}

File: services/symfony-svc/overlay/src/Entity/Payment.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
#[ORM\Table(name: 'payments')]
class Payment
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(type: 'bigint')]
    public ?string $id = null;

    #[ORM\Column(name: 'order_id', type: 'bigint', nullable: true)]
    public ?string $orderId = null;

    #[ORM\Column(name: 'customer_id', type: 'bigint', nullable: true)]
    public ?string $customerId = null;

    #[ORM\Column(name: 'amount_cents', type: 'bigint', nullable: true)]
    public ?string $amountCents = null;

    #[ORM\Column(type: 'string', nullable: true)]
    public ?string $status = null;
}
